f2/src/forms/EvidenceInfoForm.js

```diff
@@ -176,6 +173,21 @@ export const EvidenceInfoForm = (props) => {
                       <Trans id="evidencePage.addFileButton" />
                     </FileUpload>
                   </Box>
+                  <Text fontSize="md" as="span" fontWeight="bold">
```

Comment:
to get a bit more spacing below, change as="span" to as="p", or remove as="span" and change <Text> to <P>. If we make this a paragraph, it will have spacing below!

Reply:
fixed